app/src/main/res/layout-land/onboarding_slide_final.xml

```diff
@@ -76,13 +73,9 @@
         android:layout_marginStart="28dp"
         android:layout_marginTop="24dp"
         android:layout_marginEnd="52dp"
-        android:background="@drawable/primary_rounded_button"
-        android:fontFamily="sans-serif-medium"
-        android:minHeight="48dp"
         android:onClick="@{(v) -> viewModel.clickOnGetStarted()}"
         android:text="@string/get_started"
-        android:textColor="@color/white"
-        android:textSize="14sp"
+        style="@style/DefaultButton"
```

Comment:
We don't need to create this new style as of now. You can revert this change.

Reply:
Made the change!